packages/textproc/jsoncpp/package.mk

```diff
@@ -17,12 +17,12 @@
 ################################################################################
 
 PKG_NAME="jsoncpp"
-PKG_VERSION="src-0.5.0"
+PKG_VERSION="1.6.5"
 PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.kodi.tv"
-PKG_URL="http://garr.dl.sourceforge.net/project/jsoncpp/jsoncpp/0.5.0/$PKG_NAME-$PKG_VERSION.tar.gz"
+PKG_URL="http://sources.openelec.tv/devel/jsoncpp-$PKG_NAME-$PKG_VERSION.tar.yz"
```

Comment:
`yz` should be `xz`.

Although it sounds like this isn't going to happen anyway - thanks for trying! :)